cdap-support-bundle/bin/collect_support_bundle.md

```diff
@@ -0,0 +1,77 @@
+**CDAP Support Bundle Script**
+------------------------------
+
+**Description**
+---------------
+
+Collect the logs from CDAP instance deployed on Kubernetes and create a tarball file for the same logs. 
+
+**Use Case** 
+-------------
+
+This script can be used to troubleshoot issues with CDAP system services or to get information about CDAP installation. 
+
+**Command Line Arguments**
+--------------------------
+
+The CDAP Support Bundle Script takes the following command-line arguments:
+
+1. kube-namespace(n): The Kubernetes namespace where CDAP is installed.
```

Comment:
let's explicitly specify the arg names:
e.g: --kube-namepace, -n : the Kubernetes namespace where CDAP is installed

Reply:
Also specify the default values, if any, for each of these params

Reply:
Done.